<sdf version='1.6'>
    <world name='default'>
        <gui>
            <camera name='user_camera'>
                <pose frame=''>2.89 -2.13 2.55 0 0.56 2.34</pose>
            </camera>
        </gui>
        <include>
            <uri>model://sun</uri>
        </include>
        <include>
            <pose>0.0 0.0 -0.5 0 0 0</pose>
            <uri>model://my_ground_plane</uri>
        </include>
        <plugin name="stepping_plugin" filename="libstepping_plugin.so"/>
        <physics name="ode" type="ode">
            <ode>
                <constraints>
                    <contact_max_correcting_vel>0</contact_max_correcting_vel>
                </constraints>
            </ode>
            <max_step_size>0.002</max_step_size>
            <real_time_update_rate>0</real_time_update_rate>
        </physics>

        <model name="planning_scene">
            <plugin name="gazebo_ros_moveit_planning_scene" filename="libgazebo_ros_moveit_planning_scene.so">
                <topicName>gazebo/planning_scene</topicName>
                <robotName>hdt_michigan</robotName>
                <frameId>val_root</frameId>
                <updatePeriod>0.1</updatePeriod>
                <scalePrimitivesFactor>1.1</scalePrimitivesFactor>
            </plugin>
        </model>
        <plugin name="gazebo_ros_tf_plugin" filename="libgazebo_ros_tf_plugin.so">
        </plugin>
        <plugin name="collision_map_plugin" filename="libcollision_map_plugin.so"/>

        <include>
            <pose>0.0 0.64 0.115 0 0 1.5707</pose>
            <uri>model://table</uri>
        </include>

        <include>
            <pose>0.06 0.553 0.18 0 0 1.5707</pose>
            <uri>model://tissues</uri>
        </include>
        <include>
            <pose>-0.004 0.56 0.25 0 0 0</pose>
            <uri>model://level</uri>
        </include>
        <include>
            <pose>-0.52 0.555 0.11 0 0 0</pose>
            <uri>model://mustard</uri>
        </include>

    </world>
</sdf>
